pkg/ext-proc/metrics/metrics.go

```diff
@@ -0,0 +1,86 @@
+package metrics
+
+import (
+	"sync"
+	"time"
+
+	compbasemetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	klog "k8s.io/klog/v2"
+)
+
+const (
+	InferenceModelComponent = "inference_model"
+)
+
+var (
+	requestCounter = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Subsystem:      InferenceModelComponent,
+			Name:           "request_total",
+			Help:           "Counter of inference model requests broken out for each model and target model.",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"model_name", "target_model_name"},
```

Comment:
should we also have a "status" to indicate whether there is an error for the request?  And perhaps for the other metrics as well?

Reply:
We will add a request error counter metric in follow-up PRs.